runtime/domain.c

```diff
@@ -463,8 +463,10 @@ static void create_domain(uintnat initial_minor_heap_wsize) {
      set atomically */
   caml_plat_lock(&all_domains_lock);
 
-  /* wait until any in-progress STW sections end */
+  /* Wait until any in-progress STW sections end. */
   while (atomic_load_acq(&stw_leader))
+    /* [caml_plat_wait] releases [all_domains_lock] until the current
```

Comment:
Please can we put `{` `}` around that while block, it gives me the shivers.

Reply:
Done, thanks.